MetadataExtractor.Tests/DirectoryTest.cs

```diff
@@ -49,8 +49,8 @@ public void UnderlyingInt()
             Assert.Equal(value.ToString(), _directory.GetString(tagType));
             Assert.True(_directory.TryGetRational(tagType, out Rational rational));
             Assert.Equal(new Rational(value, 1), rational);
-            Assert.Equal([value], _directory.GetInt32Array(tagType));
-            Assert.Equal(new[] { unchecked((byte)value) }, _directory.GetByteArray(tagType));
+            Assert.Equal((int[])[value], _directory.GetInt32Array(tagType));
```

Comment:
I'm curious. Did Visual Studio recommend the original conversation to a collection literal? If so I'll file a big against the analyser for suggesting code that changes semantics.

Reply:
I'm on my phone so haven't investigated, but can we write this instead:


```
Assert.Equal<int>([1, 2], foo);
```

If not then maybe the original form would be more common.

```
new int[] { 1, 2 }
```

Reply:
Yep, it was recommended by VS. I agree that we may should update the analyzer rules here. 


> I'm curious. Did Visual Studio recommend the original conversation to a collection literal? If so I'll file a big against the analyser for suggesting code that changes semantics.

Yep, it was recommended by VS. I agree that we may should update the analyzer rules here.